pytorch_lightning/utilities/model_summary.py

```diff
@@ -436,40 +415,17 @@ def _is_lazy_weight_tensor(p: Tensor) -> bool:
     return False
 
 
-def summarize(
-    lightning_module: "pl.LightningModule", mode: Optional[str] = None, max_depth: Optional[int] = None
-) -> ModelSummary:
+def summarize(lightning_module: "pl.LightningModule", max_depth: Optional[int] = None) -> ModelSummary:
```

Comment:
max_depth shouldn't be Optional anymore, and the default value should be 1.

Reply:
Updated!